### PR TITLE
Deprecate curl calls for POST

### PR DESCRIPTION
**Pull request by mloviska**
Especially when using `root-console` the `curl` calls take too long to process. Use `Mojo::UserAgent` methods instead.

- Verification run: http://kepler.suse.cz/tests/20521#step/image_info/1

### CODE REVIEW

**Review by jlausuch — COMMENTED** [review]
This is so cool, very good job. Some small suggestion about record_infos

File: lib/db_utils.pm [hunk 1]
```diff
@@ -162,27 +157,32 @@ sub push_image_data_to_db {
...
+    my $data = encode_json(\%args);
+    record_info('Data', $data);
+
+    my $res = $ua->post("$url" => {Authorization => "Bearer $token", Accept => "application/json",
+            'Content-type' => "application/json"} => json => $data)->result();
+
     # if successful push, it should return 'HTTP/1.1 201 Created'
-    if ($cmd_output =~ /(?=.*201 Created)/) {
+    if ($res->code == 201) {
         record_info("DB", "Image data has been successfully pushed to the Database.");
```
[the remark this answers is not in the document]
Comment (mloviska) [reply lib/db_utils.pm:176]: we can try that out

[the remark this answers is not in the document]
Comment (mloviska) [reply lib/db_utils.pm]: done

**Review by pdostal — APPROVED** [review]
:rocket:

**Review by ricardobranco777 — APPROVED** [review]

**Comment by grisu48** [comment]
This is a very good idea, thanks @mloviska :partying_face:

**Pull request merged**